File: 12577.py
# -*- coding: utf-8 -*-
"""
Coded on Spyder

@author: Iván
"""

chant = str(input())
case = 0
while (chant != '*'):
    case+=1
    if chant == "Hajj":
        print("Case " + str(case) + ": " + "Hajj-e-Akbar" )
    else:
        print("Case " + str(case) + ": " + "Hajj-e-Asghar" )
    
    chant = str(input())

File: C++/12577.cpp
#include <iostream>

// 12577 - Hajj-e-Akbar - Iván - Visual Code Studio

using namespace std;

int main(){
    string chant = "";
    cin >> chant;
    int i = 0;
    while (chant != "*") {
        i++;
        if (chant == "Hajj") {
            cout << "Case " << i << ": Hajj-e-Akbar" << endl;
        } else if (chant == "Umrah") {
            cout << "Case " << i << ": Hajj-e-Asghar" << endl;
        }

        cin >> chant;
    }

    return 0;
}
